packages/runner/src/crawler/hc-crawler-options-factory.ts

```diff
@@ -20,19 +20,17 @@ export class HCCrawlerOptionsFactory {
         @inject(loggerTypes.Process) private readonly currentProcess: typeof process,
     ) {}
 
-    public createConnectOptions(url: string, browserWSEndpoint: string): CrawlerConnectOptions {
-        const launchOptions = this.createLaunchOptions(url);
+    public createConnectOptions(crawlUrl: string, baseUrl: string, browserWSEndpoint: string): CrawlerConnectOptions {
```

Comment:
>baseUrl [](start = 50, length = 7)

rename this to websiteUrl, to avoid confusion with 
 baseUrl: crawlUrl, line

Reply:
renamed.